www/templates/www/usersettings.html

```diff
@@ -147,6 +147,20 @@ <h1>{% trans 'NFC Card access' %}</h1>
     </tr>
   </table>
 
+  {% if show_send_email %}
+  <h1>{% trans 'Send e-mail to user' %}</h1>
+  <form action="{% url 'usersettings_send_mail' userdetails.id %}" method="post">
+    {% csrf_token %}
+    Subject:
```

Comment:
```suggestion
    {% trans 'Subject' %}:
```

Reply:
Good catch! Arguably we should localize the aria-label attribute, as well